sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -2,147 +2,180 @@
-                new ContentProperties()
+            var credential = new AzureCredentials(
+                new ServicePrincipalLoginInformation
                 {
-                    CanList = true,
-                    CanRead = true,
-                    CanWrite = true,
-                    CanDelete = true
-                });
-
-            RepositoryProperties properties = await _client.GetPropertiesAsync();
-
-            Assert.IsTrue(properties.WriteableProperties.CanList);
-            Assert.IsTrue(properties.WriteableProperties.CanRead);
-            Assert.IsTrue(properties.WriteableProperties.CanWrite);
-            Assert.IsTrue(properties.WriteableProperties.CanDelete);
-        }
-
-        public async Task InitializeTagProperties()
-        {
-            await _client.SetTagPropertiesAsync(
-                _tagName,
-                new ContentProperties()
-                {
-                    CanList = true,
-                    CanRead = true,
-                    CanWrite = true,
-                    CanDelete = true
-                });
-
-            RepositoryProperties properties = await _client.GetPropertiesAsync();
-
-            Assert.IsTrue(properties.WriteableProperties.CanList);
-            Assert.IsTrue(properties.WriteableProperties.CanRead);
-            Assert.IsTrue(properties.WriteableProperties.CanWrite);
-            Assert.IsTrue(properties.WriteableProperties.CanDelete);
+                    ClientId = TestEnvironment.ClientId,
+                    ClientSecret = TestEnvironment.ClientSecret,
+                },
+                TestEnvironment.TenantId,
+                AzureEnvironment.AzureGlobalCloud);
+
+            var _registryClient = new ContainerRegistryManagementClient(credential.WithDefaultSubscription(TestEnvironment.SubscriptionId));
+            _registryClient.SubscriptionId = TestEnvironment.SubscriptionId;
+
+            var importSource = new ImportSource
+            {
+                SourceImage = "library/hello-world",
+                RegistryUri = "registry.hub.docker.com"
+            };
+
+            await _registryClient.Registries.ImportImageAsync(
+                resourceGroupName: TestEnvironment.ResourceGroup,
+                registryName: TestEnvironment.Registry,
+                parameters:
+                    new ImportImageParameters
+                    {
+                        Mode = ImportMode.Force,
+                        Source = importSource,
+                        TargetTags = new List<string>()
+                        {
+                            $"library/hello-world:{tag}"
+                        }
+                    });
         }
 
         [RecordedTest]
         public async Task CanGetRepositoryProperties()
         {
-            RepositoryProperties properties = await _client.GetPropertiesAsync();
+            // Arrange
+            ContainerRepositoryClient client = CreateClient();
```

Comment:
Since `CreateClient` is a `[Setup]` method, it's already getting called by the test framework before each test. You should be able to make `client` a local field and have the `CreateClient` method assign the client to that for tests to use.

Reply:
I refactored it so that `CreateClient` is no longer a `[SetUp]` method.

Reply:
oh, sorry I missed that :)